Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.h

```diff
@@ -231,6 +231,9 @@ class MediaPlayerPrivateMediaStreamAVFObjC final
 
     MediaStreamTrackPrivate* activeVideoTrack() const;
 
+    LayerHostingContextID hostingContextID() const override;
+    void setVideoInlineSizeFenced(const FloatSize&, const WTF::MachSendRight&) override;
```

Comment:
We can make them final.
Ideally we would move them to the SampleBufferDisplayLayer::Client section. Strangely, this section is currently below, it could probably be changed.